### ISSUE TITLE
Incompatible with merge queues

### ISSUE THREAD

**Issue by HHK1** [13rac1/block-fixup-merge-action#31]
There is a new beta feature in GitHub called merge queues: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue

This action could be used in there, just like with Pull requests, but it's not compatible

**Comment by cooljeanius** [comment]
yeah I'm interested in starting to use the merge-queues feature more, but it's kinda hard when stuff doesn't support them much...
